cluster/images/etcd/rollback/rollback.go

```diff
@@ -325,7 +325,8 @@ func applyRequest(r *pb.Request, applyV2 etcdserver.ApplierV2) {
 		applyV2.Put(r)
 	case "DELETE":
 		applyV2.Delete(r)
-	case "POST", "QGET", "SYNC":
+	//case "POST", "QGET", "SYNC":
+	case "POST", "GET", "SYNC":
```

Comment:
I am not sure about this change. What's up with "QGET" here? Please add a test to verify this.